build.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+(
+
+if [[ " $* " =~ " "("-h"|"--help")" " ]]
+then
+  echo "Update and (re-)build backscrub"
+  echo "USAGE:"
+  echo "  ./build.sh [OPTIONS]"
+  echo ""
+  echo "  Options:"
+  echo "    --h, --help  Show this help message and exit"
+  echo "    --no-update  Skip fetching updates from github"
+  exit 0
+fi
+
+# Get the root directory of the repository
+wd="$(cd "$(dirname $0)" || exit 1; pwd)"
```

Comment:
```suggestion
wd="$(cd "$(dirname "$0")" || exit 1; pwd)"
```

Though I'm not sure how calling this script as `--foo/foo.sh` would react with the above command …